.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,85 @@
+name: update node_modules
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize, labeled]
+
+env:
+  GITHUB_BASE: rhinstaller/anaconda-webui
+
+jobs:
+  dependabot:
+    environment: npm-update
+    permissions:
+      contents: read
+      pull-requests: write
+    timeout-minutes: 5
+    # 22.04's podman has issues with piping and causes tar errors
+    runs-on: ubuntu-20.04
```

Comment:
We run this on ubuntu-latest, did you copy this from a cockpit project we forgot to update? Because I can't find any.